refaster-support/src/main/java/tech/picnic/errorprone/refaster/AnnotatedCompositeCodeTransformer.java

```diff
@@ -148,8 +148,7 @@ private static <A extends Annotation> Optional<A> getAnnotationValue(
 
   private static SeverityLevel overrideSeverity(SeverityLevel severity, Context context) {
     ErrorProneOptions options = context.get(ErrorProneOptions.class);
-    SeverityLevel minSeverity =
-        ErrorProneFork.isSuggestionsAsWarningsEnabled(options) ? WARNING : SUGGESTION;
+    SeverityLevel minSeverity = options.isSuggestionsAsWarnings() ? WARNING : SUGGESTION;
```

Comment:
Pitest flags this code as not covered, but that's not true: it's just that the tests are in another module. `RefasterTest` will fail if this line is modified.